packages/attention/src/component.tsx

```diff
@@ -1,154 +1,237 @@
+	const attentionRef = useRef<HTMLDivElement | null>(null)
+	const arrowRef = useRef<HTMLDivElement | null>(null)
+
+	const attentionState = {
+		get isShowing() {
+			return isShowing
+		},
+		get isCallout() {
+			return rest.callout
+		},
+		get actualDirection() {
+			return actualDirection
+		},
+		set actualDirection(v) {
+			setActualDirection(v)
+		},
+		get directionName() {
+			return placement
+		},
+		get arrowEl() {
+			return arrowRef.current
+		},
+		get attentionEl() {
+			return attentionRef.current
+		},
+		set attentionEl(v) {
+			attentionRef.current = v
+		},
+		get targetEl() {
+			return targetEl?.current
+		},
+		get noArrow() {
+			return props.noArrow
+		},
+	}
+
+	//TODO: See if we can move this function to the core-repo:
+	const translatedDirection = (() => {
+		switch (opposites[actualDirection]) {
+			case 'top':
+				return i18n._({
+					id: 'attention.direction.top',
+					message: 'top',
+					comment:
+						'Default screenreader message for top direction in the attention component',
+				})
+			case 'right':
+				return i18n._({
+					id: 'attention.direction.right',
+					message: 'right',
+					comment:
+						'Default screenreader message for right direction in the attention component',
+				})
+			case 'bottom':
+				return i18n._({
+					id: 'attention.direction.bottom',
+					message: 'bottom',
+					comment:
+						'Default screenreader message for bottom direction in the attention component',
+				})
+			case 'left':
+				return i18n._({
+					id: 'attention.direction.left',
+					message: 'left',
+					comment:
+						'Default screenreader message for left direction in the attention component',
+				})
+			default:
+				return ''
+		}
+	})()
+
+	// TODO: See if we can move this function to the core repo:
+	const activeAttentionType = (() => {
+		switch (true) {
+			case props.tooltip:
+				return i18n._({
+					id: 'attention.tooltip',
+					message: 'tooltip',
+					comment:
+						'Default screenreader message for tooltip in the attention component',
+				})
+			case props.callout:
+				return i18n._({
+					id: 'attention.callout',
+					message: 'callout',
+					comment:
+						'Default screenreader message for callout in the attention component',
+				})
+			case props.popover:
+				return i18n._({
+					id: 'attention.popover',
+					message: 'popover',
+					comment:
+						'Default screenreader message for popover in the attention component',
+				})
+			default:
+				return ''
+		}
+	})()
+
+	// TODO: See if we can move this function to the core repo:
+	const getDefaultAriaLabel = () => {
+		const pointingAtTranslation = i18n._({
+			id: 'attention.ariaLabel.pointingAt',
+			message: 'pointing to the',
+			comment:
+				'Default screenreader message for the pointingAt function in the attention component',
+		})
+
+		const pointingAt = !props.noArrow
+			? `${pointingAtTranslation} ${translatedDirection}`
+			: ''
+
+		return `${activeAttentionType} ${pointingAt}`
```

Comment:
Did we skip "speech bubble" as part of the label?

Reply:
I was thinking if `speech bubble` is maybe not neccessary? Do people understand if we just have `callout` or `tooltip` or `popover`? Otherwise I can put `speech bubble` back and translate it as well. :)

Reply:
In my opinion, tooltip might be self-explanatory, but popover and callout sound a bit too generic. Is it a popover menu, popover modal etc.? Based on the a11y report, it appears that if a component looks like a speech bubble (💬) it should be described as one.

Reply:
That's true! Ok I will add `speech bubble` then! :)